### DISCUSSION TITLE
Disable redirects for potential bot visitors

### DISCUSSION THREAD

**Discussion by fessnecro** [shlinkio/discussions#121] [Help wanted] [answered]
Hello! How can i disable Potential bot to being redirected to long url?

**Comment by acelaya** [comment, marked as answer]
No, that's not supported. That would fall more in the "bot blocking" area, which is out of Shlink's scope.

There are better services that you can put in front of Shlink if you want to block bots. Other than that, Shlink just tracks the fact that they could be bots, but behaves the same as with any other visitor.

**Comment by fessnecro** [comment]
Maybe it's a good idea to add that feature?
Some setting to block visiting bot?

**Reply by acelaya** [reply]
I already answered to that above 🙂

It's out of Shlink's scope
